### DISCUSSION TITLE
Ideas for How to ?

### DISCUSSION THREAD

**Discussion by Pradumnasaraf** [Pradumnasaraf/open-source-with-pradumna#31] [Ideas]
![Screenshot from 2022-05-21 18-55-29](https://user-images.githubusercontent.com/51878265/169653673-b730b941-1c43-4521-9c76-96a13f134509.png)

The aim is to build the largest Open Source guide. It will contain Step by step procedures to perform the activities like "How to create a PR", "How to Clone a repo", etc. Each topic/guide has its separate Markdown file with a detailed explanation including Text, Screenshot, Screen recording, Etc

**You can drop your suggestions and topic which you think need to be covered.**

**Comment by Pradumnasaraf** [comment]
#### How-to Topics-
- Create a PR.
- Raise and Issue.
- Add a Sponsor
- Start a discussion (Done)
- Link an issue to a PR
- Review PR
- Clone a Repo
- Add issue templates
- Add PR template
- Star a Repo
- Fork a Repo
- Lable an Issue

### Topics need to be added
Note -  For the below topics dedicated markdown file is not present you can raise an issue and work on them, even other topics would you like to add 

- Make a repo private/public
- Archive a repo
- Create a project board
- Create a release (Manual)
- Create a tag (Manual)
- Create a license
- Use GitHub pages
- Add a Social preview to a repo
- Delete this repository
- Create a GitHub secret

**Reply by Pradumnasaraf** [reply]
- Create a PAT

**Comment by KendallDoesCoding** [comment]
Ideas
- How to star a repo
- How to fork a repo
- How to watch a repo
- How to sign up for GitHub Student Plan
- How to view PR's created by you
- How to view Issues created by you

**Reply by Pradumnasaraf** [reply]
Great topic Kendall, thanks for sharing.

**Comment by ShubhamJagtap2000** [comment]
**Suggestions**

- GitHub Actions
- Why GitHub Actions?
- GitHub Code Reviews
- GitHub Projects and its benefits
- GitHub Website Hosting

**Reply by Pradumnasaraf** [reply]
Thank you for putting it here!
